### PR TITLE
add: tr.m3u Missing channels added.

### PR DESCRIPTION
**Pull request by ramazansancar**
upd: Missing information corrected.

Info: The channel added as `VTV.tr` was actually broadcasting as `KanalV.tr`.

### CODE REVIEW

**Comment by ramazansancar** [comment]
https://github.com/iptv-org/iptv/pull/17047 Required channels for broadcasts here have been added.

File: data/channels.csv [hunk 1]
```diff
@@ -31653,7 +31659,6 @@ VTV.mv,VTV,,,,MV,,,c/MV,div,,FALSE,2008-09-01,,,https://www.vnews.mv/,https://i.
 VTV.ro,VTV,,,,RO,,,c/RO,ron,,FALSE,,,,https://vtv.ro/,https://i.imgur.com/nREmG6e.png
 VTV.rs,VTV,ВТВ,,Srbija danas doo,RS,,Belgrade,c/RS,srp,,FALSE,,,,http://www.novosadska.tv/,https://i.imgur.com/7jCHU4c.png
 VTV.si,VTV,,,,SI,,,c/SI,slv,,FALSE,,,,,https://i.imgur.com/1enjDJ6.png
-VTV.tr,VTV,,,,TR,,,c/TR,tur,,FALSE,,,,,https://i.imgur.com/o26IDy4.png
```
Comment (Alstruit) [inline data/channels.csv:31656]: Unexplained removal?
https://www.vtv.net.tr/

**Review by Alstruit — CHANGES_REQUESTED** [review]

File: data/channels.csv (hunk 1 above)
Comment (ramazansancar) [reply data/channels.csv:31656]: Verify that it belongs to 'Kanal V' by looking at its image.

I mentioned in the PR statement

Comment (ramazansancar) [reply data/channels.csv:31656]: When I have time I will add it again and update the website and picture information.

**Comment by ramazansancar** [comment]
The required data correction for VTV.tr and the live broadcast link of the relevant channel have been added. FYI @Alstruit 

This commit: https://github.com/iptv-org/iptv/pull/17047/commits/77c2facddd5e61e4b63d590b64dba715b18789a6

**Review by Alstruit — DISMISSED** [review]

**Review by Dum4G — APPROVED** [review]

File: data/channels.csv [hunk 2]
```diff
@@ -12486,8 +12489,8 @@ KanalS.tr,Kanal S,,,,TR,,Samsun,c/TR,tur,news,FALSE,,,,https://www.kanals.com.tr
 KanalSim.cy,Kanal Sim,,,,CY,,,c/CY,tur,,FALSE,,,,http://www.kanalsim.com/,https://i.imgur.com/UHYJURw.png
 KanalTuran.fr,Kanal Turan,,,,FR,,Strasbourg,c/FR;c/AZ,aze,news,FALSE,,,,https://www.kanalturan.com/,https://i.imgur.com/9AWwu2o.png
 KanalUrfa.tr,Kanal Urfa,,,,TR,,,c/TR,tur,,FALSE,,,,https://kanalurfa.com/,https://i.imgur.com/EvgQmoT.png
-KanalV.tr,Kanal V,,,,TR,,,c/TR,tur,,FALSE,,,,,https://i.imgur.com/7RNjTno.png
-KanalZ.tr,Kanal Z,,,,TR,,,c/TR,tur,,FALSE,,,,,https://i.imgur.com/Kr8960d.png
+KanalV.tr,Kanal V,,,,TR,,Antalya,c/TR,tur,,FALSE,,,,https://www.kanalv.com.tr/,https://i.imgur.com/o26IDy4.png,https://i.imgur.com/7RNjTno.png
```
Comment (Dum4G) [inline data/channels.csv:12492]: why 2 imgur images?

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: data/channels.csv (hunk 2 above)
Comment (ramazansancar) [reply data/channels.csv:12492]: I'm not sure if it would be better to keep the logo according to the standard it complies with. What do you think? @Dum4G

Comment (Dum4G) [reply data/channels.csv:12492]: As long as it is up to 512x512 png image that you feel is readable to human eye I'm okay with it. But there should be only one of them, current implementation will break the database

Comment (ramazansancar) [reply data/channels.csv:12492]: I will fix the appropriate one when it becomes available.

File: data/channels.csv [hunk 3]
```diff
@@ -35037,6 +35043,7 @@ WorldPokerTour.fr,World Poker Tour,WPT,Pluto TV,,FR,,,c/FR,fra,sports,FALSE,,,,h
 WorldPokerTour.it,World Poker Tour,,Pluto TV,,IT,,,c/IT,ita,sports,FALSE,,,,https://pluto.tv/en/live-tv/world-poker-tour-it,https://i.imgur.com/e3yJ9QN.png
 WorldPokerTour.uk,World Poker Tour,,Pluto TV,,UK,,,c/UK,eng,sports,FALSE,,,,https://pluto.tv/en/live-tv/world-poker-tour-gb,https://i.imgur.com/znqxz2W.png
 WorldPokerTour.us,World Poker Tour,,,,US,,,r/INT,eng,sports,FALSE,,,,https://www.worldpokertour.tv/watch/channel/live/live/hFz9XetVSqRy-pluto,https://i.imgur.com/98kLMjj.png
+WorldTurk.tr,World Türk TV,,,,TR,,,c/TR,tur,,FALSE,,,,https://worldturk.com.tr/world-turk-tv-canli-yayin-izle/,https://www.artidijitalmedya.com/image/400x400/cdn/channels/v1/logo_151.png
```
Comment (Dum4G) [inline data/channels.csv:35046]: image
tvg-id must be WorldTurkTV.tr
channel name must be World Turk TV
alternate name must be World Türk TV

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: data/channels.csv (hunk 3 above)
Comment (Dum4G) [inline data/channels.csv:35046]:
```suggestion
WorldTurkTV.tr,World Turk TV,World Türk TV,,,TR,,,c/TR,tur,,FALSE,,,,https://worldturk.com.tr/world-turk-tv-canli-yayin-izle/,https://www.artidijitalmedya.com/image/400x400/cdn/channels/v1/logo_151.png

```

**Review by Dum4G — CHANGES_REQUESTED** [review]

**Review by Dum4G — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**